openapi-spec/swagger.yaml

```diff
@@ -200,6 +186,44 @@ paths:
         required: true
         schema:
           type: string
+      - name: limit
+        in: query
+        description: Number of query results on multiple pages
+        schema:
+          minimum: 1
+          type: integer
+          format: int32
+      - name: offset
+        in: query
+        description: Start position of the pagination query
+        schema:
+          minimum: 0
+          type: integer
+          format: int32
+      - name: sortKey
```

Comment:
usually, the backend guarantees some deterministic order...sorting is purely a UI function. any specific reason the sorting is in the API ?